templates/master.yaml.tmpl

```diff
@@ -2709,6 +2709,45 @@ storage:
       contents:
         inline: |
           VAULT_TOKEN={{ .G8SVaultToken }}
+    - path: /etc/kubernetes/manifests/k8s-api-healthz.yaml
+      filesystem: root
+      mode: 0600
+      contents:
+        inline: |
+          apiVersion: v1
+          kind: Pod
+          metadata:
+            name: k8s-api-healthz
+            namespace: kube-system
+            annotations:
+              scheduler.alpha.kubernetes.io/critical-pod: ''
+          spec:
+            hostNetwork: true
```

Comment:
Is it because it's potentially deployed before calico?

Reply:
Yes, its static pod it needs to run without k8s-api  at all